### PR TITLE
Final Submission for review

### PR DESCRIPTION
**Pull request by misalazeem**
- All features for Student A and B added
- Added readme file
- Added demo video to the readme file

### CODE REVIEW

File: src/modules/fetch_meals.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+async function fetchMeals() {
```
Comment (Ismailco) [inline src/modules/fetch_meals.js:1]: - [ ] Kindly consider using ES6 syntax through all your code, you can transform this function into using arrow function syntax instead

File: src/modules/modal-block.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+import getCommentCount from './getCommentCount';
+
+const getMealDetail = async (mealid) => {
+  const url = `https://www.themealdb.com/api/json/v1/1/lookup.php?i=${mealid}`;
+  const response = await fetch(url);
+  const recipe = (await response.json()).meals;
+  return recipe[0].strInstructions;
+};
+
+async function getDetails(mealid) {
```
Comment (Ismailco) [inline src/modules/modal-block.js:11]: - [ ] Same here, you can use arrow function syntax instead

**Review by Ismailco — CHANGES_REQUESTED** [review]
#### Hi @misalazeem, @Yordinia, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️ Nice UI
✔️ Uses Jest for testing
✔️ Uses public & integrate API
✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by osamaashraf6 — APPROVED** [review]
Hi @misalazeem 😊, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
MinionsGIF (2)

## Highlights🎉🎉

- [x] Nice design 💯
- [x] Beautiful UI🚀 
- [x] The PR has a proper title and a descriptive body 👍
- [x]  Good using the API for posting data and getting it 👌

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**